src/ddmd/root/array.d

```diff
@@ -199,12 +199,12 @@ public:
 struct BitArray
 {
 nothrow:
-    size_t length() const pure
+    @property size_t length() const pure
```

Comment:
So I understand this is a stylistic change. The problem with it being mixed in this PR is that now I worry the old code would no longer work.

Stylistic changes are best left for a separate PR where introducing regressions is not a problem. FWIW syle-wise we want to have as few property annotations as possible, not as many. So you'll have resistance from me if you tried to add property annotations without necessity.